<ui version="4.0" >
 <class>LabelPositionAssistentClass</class>
 <widget class="QDockWidget" name="LabelPositionAssistentClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Label position assistent</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>700</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab" >
       <property name="geometry" >
        <rect>
         <x>8</x>
         <y>68</y>
         <width>676</width>
         <height>406</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Step 1 (organism selection)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text" >
             <string>Select an organism</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox" />
          </item>
         </layout>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="pushButton_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Proceed to next step</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QGroupBox" name="groupBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string>Advanced options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Maximum network depth</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="lineEdit" >
             <property name="maximumSize" >
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMask" >
              <string>D999; </string>
             </property>
             <property name="text" >
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <spacer name="verticalSpacer_9" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>273</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <spacer name="verticalSpacer_8" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>273</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" >
         <spacer name="verticalSpacer_7" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>273</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>676</width>
         <height>406</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Step 2 (target metabolite selection)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Select the taget metabolite from the list below</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="0" column="2" >
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="title" >
           <string>User settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Metabolite blacklist</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit" />
           </item>
           <item>
            <widget class="QLabel" name="label_9" >
             <property name="text" >
              <string>User defined reaction properties</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit_2" />
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QListWidget" name="listWidget" />
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="pushButton" >
          <property name="text" >
           <string>Proceed to next step</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <property name="geometry" >
        <rect>
         <x>4</x>
         <y>34</y>
         <width>676</width>
         <height>406</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Step 3 (start metabolite selection)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item rowspan="5" row="0" column="0" >
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Select the start metabolite from the list below</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_2" />
          </item>
         </layout>
        </item>
        <item row="0" column="1" >
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>85</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="3" >
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title" >
           <string>Advanced options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <item>
            <widget class="QRadioButton" name="radioButton" >
             <property name="text" >
              <string>Analyze every possible tracer combination</string>
             </property>
             <property name="checked" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2" >
             <property name="text" >
              <string>Analyze only configurations with single atom labels</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="3" >
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>86</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1" colspan="3" >
         <widget class="QPushButton" name="pushButton_3" >
          <property name="text" >
           <string>Proceed to next step</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" >
         <spacer name="verticalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>85</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4" >
       <property name="geometry" >
        <rect>
         <x>4</x>
         <y>34</y>
         <width>676</width>
         <height>406</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Step 4 (results)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Analyzed tracer configurations</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="0" column="1" >
         <spacer name="verticalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item rowspan="6" row="1" column="0" >
         <widget class="QTableWidget" name="tableWidget" >
          <property name="selectionMode" >
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <column>
           <property name="text" >
            <string>Start tracer position</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Target tracer position</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string># of metabolites</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Comment</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QPushButton" name="pushButton_5" >
          <property name="text" >
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <spacer name="verticalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>135</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1" >
         <widget class="QPushButton" name="pushButton_4" >
          <property name="text" >
           <string>Show graph</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <spacer name="verticalSpacer_6" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>134</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1" >
         <widget class="QPushButton" name="pushButton_6" >
          <property name="text" >
           <string>Show metabolites</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
